stumpy/mstump.py

```diff
@@ -459,6 +492,8 @@ def _multi_distance_profile(
         μ_Q[:, query_idx],
         σ_Q[:, query_idx],
         T_subseq_isconstant,
+        Q_subseq_isconstant[:, query_idx].reshape(-1, 1),
+        query_idx=query_idx,
```

Comment:
@seanlaw 
I added this param `query_idx` so that we can pass it to `core.mass` and automatically set the self-match distances to 0. (If you think this small change needs its own PR, please let me know).